InfiniTAM/ITMLib/Trackers/Interface/ITMColorTracker.cpp

```diff
@@ -37,9 +37,11 @@ void ITMColorTracker::TrackCamera(ITMTrackingState *trackingState, const ITMView
 		minimizeLM(*this, currentPara);
 	}
 
+    const auto paraM = currentPara.GetM();
```

Comment:
Please can you replace `auto` with the actual type? We don't want to force an unnecessary C++11 dependency.

Reply:
I can do it. But according to the newest cmakefile (cmake/Flags.cmake),  C++11 is already enabled.

Reply:
It's only enabled by default on Linux - we build with it there, but it's not a compulsory dependency. The change proposed will break the build on other platforms.